.github/workflows/ci.yml

```diff
@@ -14,40 +14,37 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
-        with:
-          path: policies
-
-      - name: Clone Kyverno
-        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
-        with:
-          repository: kyverno/kyverno
-          path: kyverno
-          # The target branch of a pull request or the branch/tag of a push
-          ref: ${{ github.base_ref || github.ref_name }}
-
-      - name: Set up Go 
-        uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753
-        with:
-          go-version: "1.20"
-
-      - name: Test Policy
-        run: go run ./cmd/cli/kubectl-kyverno test ../policies
-        working-directory: kyverno
-    
       - name: Checkout kyverno/policies
-        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+        uses: actions/checkout@v2
         with:
           repository: kyverno/policies
-      
+
       - name: Validate all policies
         run: |
           #!/bin/bash
           set -euo pipefail
 
+          # Define the directory to ignore
+          ignore_dir="$GITHUB_WORKSPACE/pod-security/enforce"
```

Comment:
This isn't what I meant. We don't want to ignore a specific directory, we want to exempt any file named `kustomization.yaml`, regardless of where it's located, from being caught by this logic.